package.json

```diff
@@ -35,14 +35,14 @@
   "homepage": "https://github.com/vacuumlabs/cardano-hw-cli#readme",
   "dependencies": {
     "@babel/runtime": "^7.11.2",
-    "@cardano-foundation/ledgerjs-hw-app-cardano": "https://github.com/vacuumlabs/ledgerjs-cardano-shelley/releases/download/v2.0.2-rc.5/cardano-foundation-ledgerjs-hw-app-cardano-v2.0.2-rc.5.tgz",
+    "@cardano-foundation/ledgerjs-hw-app-cardano": "https://github.com/vacuumlabs/ledgerjs-cardano-shelley/releases/download/v2.2.0-beta/cardano-foundation-ledgerjs-hw-app-cardano-2.2.0-beta.tgz",
     "@ledgerhq/hw-transport-node-hid": "^5.25.0",
     "argparse": "^2.0.1",
     "bignumber": "^1.1.0",
     "borc": "^2.1.2",
     "cardano-crypto.js": "^5.3.6-rc.6",
     "rw": "1.3.3",
-    "trezor-connect": "https://github.com/vacuumlabs/connect/releases/download/8.1.18-rc.2/trezor-connect-8.1.18-rc.2-extended.tgz"
+    "trezor-connect": "8.1.20-extended"
```

Comment:
you can already switch to public 8.1.22 release

Reply:
Seems like that version introduced new bug. I'm getting `Error: Parameter "tokenBundle" has invalid type. "array" expected.` when `tokenBundle` is `null` or `undefined`. I can either keep it at version 8.1.20 or add the `removeNullFields` to `prepareOutput` function. @refi93

Reply:
For now I'll increase the version and add `removeNullFields`.